grafana-plugin/src/components/GTable/GTable.tsx

```diff
@@ -140,19 +140,25 @@ export const GTable = <RT extends DefaultRecordType = DefaultRecordType>(props:
 
   return (
     <div className={cx('root')} data-testid="test__gTable">
-      <Table<RT>
-        expandable={expandable}
-        rowKey={rowKey}
-        className={cx('filter-table', className)}
-        columns={columns}
-        data={data}
-        showHeader={showHeader}
-        {...restProps}
-      />
-      {pagination && (
-        <div className={cx('pagination')}>
-          <Pagination hideWhenSinglePage currentPage={page} numberOfPages={numberOfPages} onNavigate={onNavigate} />
-        </div>
+      {loading ? (
```

Comment:
do we really need to display just loading placeholder during loading stage? In general we are trying to avoid such...

Reply:
I'm going to agree on the solution with Raphael. Still I think we need some indicator because for example on ops filtering by `acknowledgment` might take even 10-15secs. Without any indication, user might think that it's just not working and until receives response might refresh the page

Reply:
we should display some indicator, but shouldn't hide the whole table